### DISCUSSION TITLE
Add a title to the picker

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [helix-editor/helix#5788] [Ideas]
The picker is entirely void of context. 

You have no indication:

- what kind of picker it is at all
- what the searched symbol is if it is a "show references" (LSP) picker. (`gi`)
- which base dir it is searching from Space-f/F
- etc.

Adding a picker title field and fill it with an appropriate string for each picker use could really help give some orientation.

**Comment by CptPotato** [comment]
I would prefer this to be an option. I usually know what I get when using each specific picker and prefer the more minimal aesthetic.

I even thought about styling the picker to be border-less (or using a single collapsed border in the center).
I've been using a more minimal window style for a while now and quite like it:

<details>
<summary>screenshot</summary>

![image](https://user-images.githubusercontent.com/3957610/216600377-a587347c-3b02-4533-8089-7dd768edb3b5.png)
</details>
